db/seeds_sandbox.rb

```diff
@@ -0,0 +1,1038 @@
+        email: Faker::Internet.email,
+        firstname: Faker::Name.first_name,
+        surname: Faker::Name.last_name,
+        password: pwd,
+        password_confirmation: pwd,
+        org: Org.find_by(abbreviation: 'OEO'),
+        language: default_language,
+        perms: [],
+        accept_terms: true,
+        api_token: Faker::Lorem.word,
+        confirmed_at: 3.month.ago,
+        created_at: 3.month.ago,
+        active:1
+    }
+    User.create(user)
+  end
+  (1..50).each do |index|
+    pwd = Faker::Lorem.unique
+    user = {
+      email: Faker::Internet.email,
+      firstname: Faker::Name.first_name,
+      surname: Faker::Name.last_name,
+      password: pwd,
+      password_confirmation: pwd,
+      org: Org.find_by(abbreviation: 'IEO'),
+      language: default_language,
+      perms: [],
+      accept_terms: true,
+      api_token: Faker::Lorem.word,
+      confirmed_at: 4.month.ago,
+      created_at: 4.month.ago,
+      active:1
+    }
+    User.create(user)
+  end
+
+
+  # Guidance Groups for the default org, funder admin's org and institution admin's org
+  # Guidance Groups are created 4 years ago
+  # -------------------------------------------------------
+  guidance_groups = [
+    {name: "Generic Guidance (provided by the example curation centre)",
+     org: Org.find_by(abbreviation: Rails.configuration.x.organisation.abbreviation),
+     optional_subset: true,
+     published: true,
+     created_at: 4.years.ago},
+    {name: "Government Agency Advice (Organisation specific guidance)",
+     org: Org.find_by(abbreviation: 'OEO'),
+     optional_subset: false,
+     published: true,
+     created_at: 4.years.ago},
+    {name: "Institution Advice (Institution specific guidance)",
+    org: Org.find_by(abbreviation: 'IEO'),
+    optional_subset: false,
+    published: true,
+    created_at: 4.years.ago}
+  ]
+  guidance_groups.each do |gg|
+    guidance_group = GuidanceGroup.find_or_create_by(org: gg[:org])
+    guidance_group.update!(gg)
+  end
+  # 5 test random guiance group
+  (1..5).each do |index|
+      gg = {
+          name: Faker::Lorem.word,
+          org: Org.all.sample,
+          optional_subset: true,
+          published: true,
+          created_at: 4.years.ago
+      }
+      guiance_group = GuidanceGroup.find_or_create_by(org: gg[:org])
+      guiance_group.update!(gg)
+  end
+  
+  # Guidances for random guidance group using random theme
+  # -------------------------------------------------------
+  (1..50).each do |index|
+    guidance = 
+      { text: Faker::Lorem.word,
+        guidance_group: GuidanceGroup.all.sample,
+        published: true,
+        themes:  [Theme.all.sample],
+        created_at: 4.years.ago
+      }
+    Guidance.create(guidance)
+    end
+
+  # Three standard templates for default org, funder admin's org and insitutional admin's org
+  # Templates are created 3 years ago
+  # -------------------------------------------------------
+  templates = [
```

Comment:
We should add at least one of DMP Assistant's default Portage templates. It will be a bit of work but it will be very beneficial to the people testing the application.

You can try following the approach outlined [here](https://medium.com/railstips/create-seed-file-from-data-already-in-database-in-rails-660e5ab92b5).

Let me know if you have any questions.